corehq/apps/locations/templates/locations/manage/location.html

```diff
@@ -126,6 +145,108 @@
 
       </div>
 
+      <div class="tab-pane {% if form_tab == "descendants" %}active{% endif %}" id="descendants-info">
```

Comment:
Pretty much reused the code in [locations.html](https://github.com/dimagi/commcare-hq/blob/cca0977071eaea50c46e1f63b9064794646ddce1/corehq/apps/locations/templates/locations/manage/locations.html#L148).